web/servlet/home/Home_de.html

```diff
@@ -66,6 +66,12 @@
                                 <!-- should have open/download button as appropriate -->
                                 <a class="btn btn-default" href="http://www.withrottle.com" role="button">Erzähle weiter »</a>
                             </p>
+                            <h4><a href="https://www.microsoft.com/store/apps/9NBLGGH4VF5J">Train Throttle</a></h4>
+                            <p><em>Train Throttle</em> is een native gashendel voor de Windows. Het is geoptimaliseerd voor tablets en mobiel.</p>
```

Comment:
@silverailscolo quite right, I ran it through google translate. I figured a botched translation would be better than nothing. Perhaps not. I'm going to try this again, to fix things. If someone with write privileges ( @rhwood ?) could reject this I'll clean up my grammar and at least get the language correct.